README.md

```diff
@@ -1,4 +1,4 @@
-# cloud-platform-environments
+q# cloud-platform-environments
```

Comment:
@keith-h would you be able to fix this typo introduced here?

Reply:
sorry - now pushed

Reply:
LGTM